subprojects/internal-performance-testing/src/main/groovy/org/gradle/performance/AbstractCrossVersionGradleProfilerPerformanceTest.groovy

```diff
@@ -64,6 +64,7 @@ class AbstractCrossVersionGradleProfilerPerformanceTest extends Specification {
 
     def setup() {
         def debugArtifactsDirectory = new File(System.getProperty(DEBUG_ARTIFACTS_DIRECTORY_PROPERTY_NAME))
+        debugArtifactsDirectory.mkdirs()
```

Comment:
Hmm... Shouldn't the task create the directory and not the test? What about the cross build and tooling api performance tests? Wouldn't they also need to create this directory?

Reply:
Removed here. Previously I mistakenly removed `@OutputDirectory` so this was not created automatically...